pkg/cmd/taskrun/delete.go

```diff
@@ -176,6 +179,25 @@ func deleteTaskRuns(s *cli.Stream, p cli.Params, trNames []string, opts *options
 		d = deleter.New("TaskRun", func(taskRunName string) error {
 			return actions.Delete(trGroupResource, cs.Dynamic, cs.Tekton.Discovery(), taskRunName, p.Namespace(), metav1.DeleteOptions{})
 		})
+		for _, trNane := range trNames {
```

Comment:
we need to do the same in deleteAllNs case also

Reply:
Yes, deleteAllNs calls allTaskRunNames() which filter outs the taskruns having pipelinruns running